pipelines/enable-resource-locks.yaml

```diff
@@ -12,15 +12,35 @@ schedules:
         - master
       always: true
 
-jobs:
-  - job: Enable_Resource_Locks
-    pool:
-      vmImage: 'ubuntu-latest'
-    steps:
-    - task: AzureCLI@2
-      displayName: 'Enable Resource locks'
-      inputs:
-        scriptType: bash
-        scriptLocation: scriptPath
-        azureSubscription: '$(azureSubscription)'
-        scriptPath: scripts/enable-resource-locking.sh
+parameters:
+  - name: subscriptions
+    type: object
```

Comment:
I really dislike this UX but i'm not sure if there's a better way 😢 

you could maybe have a variable that has the same name as the parameter that returns the service connection name? or a shell script?

might be worth it just for an experiment as we repeat this pattern quite a bit

![image](https://user-images.githubusercontent.com/21194782/144395029-901cd0aa-2d8c-425f-b08c-2d98886c3ba7.png)

Reply:
It should be only ran on a schedule and never manually as far as I was understanding it, so I guess no-one should ever really see it? Granted, it is unsightly all the same.

I'll have a look to see if there is some way it can be tidied up.

Reply:
sure but I assume you need something similar for the disable pipeline too?

Reply:
The disable pipeline will show as a dropdown for the user to pick a sub
![image](https://user-images.githubusercontent.com/80516065/144399191-b686cc41-4b54-4290-aff0-fb014d4d0063.png)